dxr/query.py

```diff
@@ -326,6 +327,62 @@ def generic_visit(self, node, visited_children):
         return visited_children or node
 
 
+# TODO next next: write some tests
+class FilterAggregator(Filter):
+    """Filter class that acts by constructing the union of some subset of the
+    filters of all currently enabled plugins.
+
+    """
+
+    def __init__(self, term, plugins, condition=None):
+        super(FilterAggregator, self).__init__(term, plugins)
+        self.filters = [f(term, plugins) for f in
+                        some_filters_by_name(plugins, condition)]
+
+    def filter(self):
+        # OR together all the underlying filters.
+        return {'or': filter(None, (f.filter() for f in self.filters))}
+
+    def highlight_content(self, result):
+        # Union all of our underlying filters.
+        return chain.from_iterable(
+            f.highlight_content(result) for f in self.filters)
+
+
+class IdFilter(FilterAggregator):
+    """Case-sensitive filter for exact matching on declarations."""
+
+    name = 'id'
+    domain = LINE
+    description = Markup('Definition: <code>id:var</code>.')
```

Comment:
I think you forgot to describe the meaning of the filter (here and in RefFilter).